deploy/lib/createContainers.js

```diff
@@ -126,6 +127,6 @@ module.exports = {
     this.applyDomainsContainer(foundContainer.id, container.custom_domains);
 
     return this.updateContainer(foundContainer.id, params)
-      .then(response => Object.assign(response, { directory: container.directory }));
+      .then((response) => Object.assign(response, { directory: container.directory }));
```

Comment:
Unnecessary change I guess.

Reply:
linter, removed it 👌  ( `Arrow functions can omit parentheses when they have exactly one parameter. In all other cases the parameter(s) must be wrapped in parentheses. This rule enforces the consistent use of parentheses in arrow functions.` )